plugins/modules/k8s.py

```diff
@@ -179,17 +179,12 @@
     state: present
     definition: "{{ lookup('file', '/testing/deployment.yml') | from_yaml }}"
 
-- name: Read definition file from the Ansible controller file system after Jinja templating
```

Comment:
Do we still want to document this as an alternative?

Reply:
I wanted to encourage users to use the new feature. Also, the example above sort of shows that.